pkg/abstractions/image/image.go

```diff
@@ -90,8 +91,10 @@ func (is *RuncImageService) VerifyImageBuild(ctx context.Context, in *pb.VerifyI
 func (is *RuncImageService) BuildImage(in *pb.BuildImageRequest, stream pb.ImageService_BuildImageServer) error {
 	log.Printf("incoming image build request: %+v", in)
 
+	baseImageTag := is.config.ImageService.Runner.Tags[in.PythonVersion]
```

Comment:
This doesn't look right... I thought we fixed this already? Shouldn't we check to see if the string is empty or if the ok/exists value is false?

Reply:
I think we don't check here because we already checked in the verify above. I just followed what is in the main branch: https://github.com/beam-cloud/beta9/blob/main/pkg/abstractions/image/image.go#L94

Reply:
Oh right. Its on the API side of things. So why did we change this at all then?

Reply:
Yeah I should revert. I had different logic and then didn't put it back in the right place when I undid all that